cli/index.js

```diff
@@ -22,8 +25,19 @@ if (!isMACValid(mac)) {
     process.exit(1);
 }
 
-wake(mac, {
-    ip: program.ip,
-    host: program.host,
-    port: program.port
-});
+if (program.relay) {
+    relay(mac, {
+        ip: program.ip,
+        host: program.host,
+        port: program.port,
+        relayport: program.relayport
+    });
+} else {
+    wake(mac, {
```

Comment:
as i mention before, we can use the same `wake` method and pass `relay` and `relayport` optiosn

Reply:
I can refactor to a single method.  I thought that the wake vs. relay were significantly different that they merited distinct methods, but I'm happy to change that.